src/go/k8s/api/redpanda/v1alpha2/user_types.go

```diff
@@ -0,0 +1,209 @@
+package v1alpha2
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func init() {
+	SchemeBuilder.Register(&User{}, &UserList{})
+}
+
+// User defines the CRD for a Redpanda user.
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:path=users
+// +kubebuilder:resource:shortName=rpu
+// +kubebuilder:printcolumn:name="Synced",type="string",JSONPath=`.status.conditions[?(@.type=="Synced")].status`
```

Comment:
Do you think it's worth it to make a constant for Synced?

Reply:
Made this and `Pending` constants.